exoplayer-codelab-01/build.gradle

```diff
@@ -1,10 +1,9 @@
 apply from: "${project.rootDir}/common.gradle"
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'com.google.android.exoplayer:exoplayer-core:2.12.0'
-    implementation 'com.google.android.exoplayer:exoplayer-dash:2.12.0'
-    implementation 'com.google.android.exoplayer:exoplayer-ui:2.12.0'
-}
\ No newline at end of file
+    implementation 'androidx.appcompat:appcompat:1.4.0'
+    implementation 'com.google.android.exoplayer:exoplayer-core:2.16.1'
```

Comment:
Unfortunately it's not possible to upgrade from ExoPlayer 2.12 to 2.16 without making some code changes. Please could we revert these version updates.